Added steps for simulating robot code w/ kotlin (#10)
Kotlin-for-FRC/kotlin-for-frc.github.io: docs/getting-started/running.md

   ![Deploy Run Configuration](../img/deploy_config.png)

3. Click on the run button in the top right corner of the IntelliJ window to deploy your robot
   program to the robot.

## Simulating Robot Code

> FRC code made with kotlin can be simulated on your computer, just like robot code written with Java, Python, or C++. To do so,either follow the "Deploying via IntelliJ" or "Deploying manually via the command line" instructions above, but replace the "deploy" keyword with the "simulateJava" keyword(I.E `./gradlew simulateJava` on Windows Powershell or a Unix shell).

Reviewer: ```suggestion
FRC code made with kotlin can be simulated on your computer, just like robot code written with Java, Python, or C++. To do so, either follow the "Deploying via IntelliJ" or "Deploying manually via the command line" instructions above, but replace the "deploy" task with the "simulateJava" task (i.e `./gradlew simulateJava` on Windows Powershell or a Unix shell).
```

> FRC code made with kotlin can be simulated on your computer, just like robot code written with Java, Python, or C++. To do so,either follow the "Deploying via IntelliJ" or "Deploying manually via the command line" instructions above, but replace the "deploy" keyword with the "simulateJava" keyword(I.E `./gradlew simulateJava` or `./gradlew.bat simulateJava` on windows).

Reviewer 2: `gradlew.bat` should execute as gradlew on windows. Executing scripts behaves differently between cmd and pwsh so we should specify which one is used.
```suggestion
FRC code made with kotlin can be simulated on your computer, just like robot code written with Java, Python, or C++. To do so,either follow the "Deploying via IntelliJ" or "Deploying manually via the command line" instructions above, but replace the "deploy" keyword with the "simulateJava" keyword(I.E `./gradlew simulateJava` on Windows Powershell or a Unix shell).
```

> WPILib's simulation allows you to run your code locally on your computer(without an actual robot). This should be used in junction with the WPILib simulation classes to simulate your mechanisms; read more about the physics simulation API [here](

Reviewer: ```suggestion
WPILib's simulation allows you to run your code locally on your computer, without an actual robot. This should be used in conjunction with the WPILib simulation classes to simulate your mechanisms; read more about the physics simulation API [here](
```

> WPILib's simulation allows you to run your code locally on your computer(without an actual robot). This should be used in junction with the WPILib simulation classes to simulate your mechanisms; read more about the physics simulation API [here](

Reviewer 2: Maybe also discuss hardware sim

Author: I feel like hardware sim is already explained pretty well in the wpilib docs(I don't think kotlin does anything better for hardware sim than java does).

Author: Personally i think that wpilib should endorse teams using abstract classes/io implementations/vendor-specific sim support over EncoderSim and such because those sim classes are only available for a limited amount of wpilib classes and aren't that great of an API to begin with
